From 385116a9adb6cb1987ac2c6e6d933610e074b710 Mon Sep 17 00:00:00 2001
From: Aaron Plattner <aplattner@nvidia.com>
Date: Tue, 3 Sep 2019 11:10:23 -0700
Subject: [PATCH 1/3] doc: Use /** @} */ to close Doxygen groups

Doxygen looks for the sequence /**, so the existing group closing markers that
use the pattern /*@}*/ are ignored. Use /** @} */ instead.

Fixes a Doxygen warning:

 libvdpau/include/vdpau/vdpau.h:4870: warning: end of file while inside a group
 libvdpau/include/vdpau/vdpau_x11.h:179: warning: end of file while inside a group

Also fixes an imbalance in the Doxgen generated module list. From

 * Core API
   * Basic Types
     * Miscellaneous Types
       * Error Handling
         * [etc.]

to

 * Core API
   * Basic Types
   * Miscellaneous Types
   * Error Handling
   * [etc.]
---
 include/vdpau/vdpau.h     | 32 ++++++++++++++++----------------
 include/vdpau/vdpau_x11.h |  4 ++--
 2 files changed, 18 insertions(+), 18 deletions(-)

diff --git a/include/vdpau/vdpau.h b/include/vdpau/vdpau.h
index 68a814d..a56643e 100644
--- a/include/vdpau/vdpau.h
+++ b/include/vdpau/vdpau.h
@@ -812,7 +812,7 @@ extern "C" {
  */
 typedef int VdpBool;
 
-/*@}*/
+/** @} */
 
 /**
  * \defgroup misc_types Miscellaneous Types
@@ -1183,7 +1183,7 @@ typedef struct {
     float alpha;
 } VdpColor;
 
-/*@}*/
+/** @} */
 
 /**
  * \defgroup error_handling Error Handling
@@ -1342,7 +1342,7 @@ typedef char const * VdpGetErrorString(
     VdpStatus status
 );
 
-/*@}*/
+/** @} */
 
 /**
  * \defgroup versioning Versioning
@@ -1413,7 +1413,7 @@ typedef VdpStatus VdpGetInformationString(
     char const * * information_string
 );
 
-/*@}*/
+/** @} */
 
 /**
  * \defgroup VdpDevice VdpDevice; Primary API object
@@ -1443,7 +1443,7 @@ typedef VdpStatus VdpDeviceDestroy(
     VdpDevice device
 );
 
-/*@}*/
+/** @} */
 
 /**
  * \defgroup VdpCSCMatrix VdpCSCMatrix; CSC Matrix Manipulation
@@ -1550,7 +1550,7 @@ typedef VdpStatus VdpGenerateCSCMatrix(
     VdpCSCMatrix *   csc_matrix
 );
 
-/*@}*/
+/** @} */
 
 /**
  * \defgroup VdpVideoSurface VdpVideoSurface; Video Surface object
@@ -1758,7 +1758,7 @@ typedef VdpStatus VdpVideoSurfacePutBitsYCbCr(
     uint32_t const *     source_pitches
 );
 
-/*@}*/
+/** @} */
 
 /**
  * \defgroup VdpOutputSurface VdpOutputSurface; Output Surface object
@@ -2063,7 +2063,7 @@ typedef VdpStatus VdpOutputSurfacePutBitsYCbCr(
     VdpCSCMatrix const * csc_matrix
 );
 
-/*@}*/
+/** @} */
 
 /**
  * \defgroup VdpBitmapSurface VdpBitmapSurface; Bitmap Surface object
@@ -2210,7 +2210,7 @@ typedef VdpStatus VdpBitmapSurfacePutBitsNative(
     VdpRect const *      destination_rect
 );
 
-/*@}*/
+/** @} */
 
 /**
  * \defgroup VdpOutputSurfaceRender VdpOutputSurface Rendering Functionality
@@ -2487,7 +2487,7 @@ typedef VdpStatus VdpOutputSurfaceRenderBitmapSurface(
     uint32_t                                 flags
 );
 
-/*@}*/
+/** @} */
 
 /**
  * \defgroup VdpDecoder VdpDecoder; Video Decoding object
@@ -3570,7 +3570,7 @@ typedef VdpStatus VdpDecoderRender(
     VdpBitstreamBuffer const * bitstream_buffers
 );
 
-/*@}*/
+/** @} */
 
 /**
  * \defgroup VdpVideoMixer VdpVideoMixer; Video Post-processing and Compositing object
@@ -4371,7 +4371,7 @@ typedef VdpStatus VdpVideoMixerRender(
     VdpLayer const *              layers
 );
 
-/*@}*/
+/** @} */
 
 /**
  * \defgroup VdpPresentationQueue VdpPresentationQueue; Video presentation (display) object
@@ -4614,7 +4614,7 @@ typedef VdpStatus VdpPresentationQueueQuerySurfaceStatus(
     VdpTime *                    first_presentation_time
 );
 
-/*@}*/
+/** @} */
 
 /**
  * \defgroup display_preemption Display Preemption
@@ -4689,7 +4689,7 @@ typedef VdpStatus VdpPreemptionCallbackRegister(
     void *               context
 );
 
-/*@}*/
+/** @} */
 
 /**
  * \defgroup get_proc_address Entry Point Retrieval
@@ -4851,8 +4851,8 @@ typedef VdpStatus VdpGetProcAddress(
     void * *  function_pointer
 );
 
-/*@}*/
-/*@}*/
+/** @} */
+/** @} */
 
 /**
  * \defgroup api_winsys Window System Integration Layer
diff --git a/include/vdpau/vdpau_x11.h b/include/vdpau/vdpau_x11.h
index edbc275..26b6090 100644
--- a/include/vdpau/vdpau_x11.h
+++ b/include/vdpau/vdpau_x11.h
@@ -167,8 +167,8 @@ typedef VdpStatus VdpPresentationQueueTargetCreateX11(
 /** \hideinitializer */
 #define VDP_FUNC_ID_PRESENTATION_QUEUE_TARGET_CREATE_X11 (VdpFuncId)(VDP_FUNC_ID_BASE_WINSYS + 0)
 
-/*@}*/
-/*@}*/
+/** @} */
+/** @} */
 
 #ifdef __cplusplus
 }
-- 
2.11.0

